pythonSynapseUtils/annotationsYaml.py

```diff
@@ -0,0 +1,137 @@
+import yaml
+
+
+def checkAgainstDict(annotDict, project, syn){
+	''' Compares annotations in use in a project against those specified by a dictionary and prints the ones that do not match the dictionary.'''
+	yamlEnt = syn.get(annotDict)
+	annotations = yaml.load(file(yamlEnt.path))
```

Comment:
Abstract this part - specify the structure of annotations, and allow user to get it in (then my tables-based version could use this as well)